### PR TITLE
ci(generative-ai): Add noxfile_config.py to all subdirectories for `generative_ai` to only run affected directory tests

### PR DESCRIPTION
**Pull request by holtskinner**
## Description

Should resolve repeated `ResourceExhausted: 429 Resource exhausted` errors

Fixes: https://github.com/GoogleCloudPlatform/python-docs-samples/issues/12672

### CODE REVIEW

File: generative_ai/batch_predict/noxfile_config.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+# You can copy this file into your directory, then it will be imported from
+# the noxfile.py.
+
+# The source of truth:
+# https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/noxfile_config.py
+
+TEST_CONFIG_OVERRIDE = {
+    # You can opt out from the test for specific Python versions.
+    "ignored_versions": ["2.7", "3.7", "3.9", "3.10", "3.12"],
```
Comment (iennae) [inline generative_ai/batch_predict/noxfile_config.py:25]: Can we have 3.8 and 3.12? (remove 3.12 from the ignored versions)

Comment (msampathkumar) [reply generative_ai/batch_predict/noxfile_config.py:25]: 3.8 is also deprecated! Should we switch to `3.9` & `3.12` ?

https://endoflife.date/python

Comment (holtskinner) [reply generative_ai/batch_predict/noxfile_config.py:25]: Done

**Review by msampathkumar — APPROVED** [review]
LGTM.

**Review by iennae — APPROVED** [review]
Thank you!

**Pull request merged**